include/lightweight_filtering/SigmaPoints.hpp

```diff
@@ -174,7 +174,7 @@ class SigmaPoints{
     computeFromGaussian(identity,P);
   };
   const mtState& operator()(unsigned int i) const{
-    assert(i<L_);
+    CHECK(i<L_);
```

Comment:
would be nice to use CHECK_LT

Reply:
as discussed, not changing checks to make merging back to main rovio easier